tor/onionfile/onion_file_test.go

```diff
@@ -0,0 +1,97 @@
+package onionfile
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+)
+
+var (
+	privateKey = []byte("RSA1024 hide_me_plz")
+)
+
+type MockEncrypter struct{}
+
+func (m MockEncrypter) EncryptPayloadToWriter(_ bytes.Buffer, _ io.Writer,
+	_ []byte) error {
+
+	return nil
+}
+
+func (m MockEncrypter) DecryptPayloadFromReader(_ io.Reader, _ []byte) ([]byte,
+	error) {
+
+	return privateKey, nil
+}
+
+// TestOnionFile tests that the OnionFile implementation of the OnionStore
```

Comment:
nit: Change `OnionFile` to either `onionFile` to match the package name, or `File` to match the new struct name.